### DISCUSSION TITLE
Deploy static website to S3 bucket

### DISCUSSION THREAD

**Discussion by RebootGG** [coollabsio/coolify#3459] [Feature Requests]
It would be really nice to be able to deploy static websites to S3 buckets.

Most S3 bucket have a "website" feature nowadays, eg scaleway: https://www.scaleway.com/en/docs/storage/object/how-to/use-bucket-website/

It's the cheapest way to serve a simple website ! And a lot of people choose Coolify to avoid unjustified costs.

**Comment by melang982** [comment]
That would also be very useful for those of us who intend to serve frontend from S3, and backend from a VPS. I'd love to manage and deploy it all from one place (Coolify).

**Comment by JSanchezFDZ** [comment]
One of the best features Coolify could have.
